ecf/scripts/gdas/wave/prep/jgdas_wave_prep.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:10:00
-#PBS -l place=vscatter,select=1:mpiprocs=128:ompthreads=1:ncpus=128
+#PBS -l place=vscatter,select=1:mpiprocs=5:ompthreads=1:ncpus=5
 #PBS -l place=excl
```

Comment:
Does this job still need exclusive use of a node due to memory?

Reply:
I don't know exactly what the memory use is in this job, hence the exclusive use.